autoflip_calculators/text_detection_calculator.cc

```diff
@@ -240,7 +239,7 @@ ::mediapipe::Status TextDetectionCalculator::Process(
     region->mutable_signal_type()->set_standard(SignalType::TEXT);
 
     // Score the text based on image cues.
-    if (options_.use_visual_scorer()) {
+    if (use_visual_scorer_) {
```

Comment:
Instead of making this a member var, how about just use from the options directly:
if(options_.use_visual_scorer())